spec/features/bookmarks_spec.rb

```diff
@@ -28,6 +28,7 @@
    visit topic_path(@topic)
    click_link("Show")
    click_link("Edit Bookmark")
+   expect(page).to have_field("bookmark_url", with: "valid url", type: "url")
    fill_in "bookmark_url", with: "www.google.com"
```

Comment:
The app ought to be able to flexibly handle URLs entered in the following formats:

- `www.google.com` (no scheme prefix)
- `https://www.google.com` (with scheme prefix)

This applies for URLs submitted by email and URLs submitted through the frontend UI.

Write tests to ensure the app can handle these situations. The bookmark links displayed in the view source of the apps pages should have the scheme prefix:

```html
<!-- Incorrect -->
<a href="www.google.com">...</a>

<!-- Correct -->
<a href="http://www.google.com">...</a>
```